docs/DeveloperGuide.md

```diff
@@ -141,7 +150,11 @@ for subsequent prompts.
 
 ---
 
-### Create New Workout
+### Exercise
+
+---
+### Workout
```

Comment:
I think to be safe, can consider adding an extra line between these two lines to avoid any potential rendering issues.